### PR TITLE
disable refund admin page

### PR DESCRIPTION
**Pull request by ahsan-ul-haq**
LEARNER-4681
FYI: @edx/learner @edx/learner-spartans

### CODE REVIEW

**Review by staubina — APPROVED** [review]
The changes seem straightforward to me.

**Comment by staubina** [comment]
We should have at least one more reviewer on this PR.

**Review by Ayub-Khan — APPROVED** [review]
LGTM

File: ecommerce/conf/locale/en/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -7,8 +7,8 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: ecommerce\n"
-"Report-Msgid-Bugs-To: openedx-translation@googlegroups.com\n"
```
Comment (Ayub-Khan) [inline ecommerce/conf/locale/en/LC_MESSAGES/django.po:10]: Im not sure why those are being removed, but put them back. 'Report-Msgid-Bugs-To: openedx-translation@googlegroups.com\n'

File: ecommerce/conf/locale/en/LC_MESSAGES/djangojs.po [hunk 2]
```diff
@@ -7,8 +7,8 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: ecommerce\n"
-"Report-Msgid-Bugs-To: openedx-translation@googlegroups.com\n"
```
Comment (Ayub-Khan) [inline ecommerce/conf/locale/en/LC_MESSAGES/djangojs.po:10]: Im not sure why those are being removed, but put them back. 'Report-Msgid-Bugs-To: openedx-translation@googlegroups.com\n'

**Review by Ayub-Khan — CHANGES_REQUESTED** [review]

**Review by Ayub-Khan — APPROVED** [review]

**Pull request merged**